models/sipnet/R/sample.IC.SIPNET.R

```diff
@@ -7,38 +7,39 @@
 # http://opensource.ncsa.illinois.edu/license.html
 #-------------------------------------------------------------------------------
 
-## samples intial conditions for SIPNET
-##' @title sample.IC.SIPNET
-##' @name  sample.IC.SIPNET
-##' @author Mike Dietze and Ann Raiho
-##' 
-##' @param ne number of ensembles
-##' @param state state variables you want to pull
-##' @description samples intial conditions for SIPNET
-##' 
-##' @return IC matrix of initial conditions
-##' @export
-##' 
+#' Sample initial conditions for SIPNET
```

Comment:
Is this function still used in the workflow? Seems like IC files should now be based on data and generated by model-agnostic workflows (e.g. ic_process)